src/color/opacify.js

```diff
@@ -12,23 +12,23 @@ import curry from '../internalHelpers/_curry'
  * // Styles as object usage
  * const styles = {
  *   background: opacify(0.1, '#fff');
- *   background: opacify(0.2, 'hsl(0, 0%, 100%)'),
- *   background: opacify(0.5, 'rgba(255, 0, 0, 0.8)'),
+ *   background: opacify(0.2, 'hsl(0, 0%, 20%)'),
```

Comment:
can you change it to `opacify(0.2, 'hsla(0, 0%, 100%, 0.5)')`
results in: rgba(255,255,255,0.7)